src/database/effects/cards/2-2-060.ts

```
@@ -0,0 +1,50 @@
+        { id: '1', description: '紫ゲージ+1' },
+        { id: '2', description: 'デッキからインターセプトをセット' },
+      ],
+    });
+
+    switch (choice) {
+      case '1':
+      default:
+        await System.show(stack, '選略・魔導の心得', '紫ゲージ+１');
+        stack.core.room.soundEffect('purple-increase');
+        // TODO: 紫ゲージを実装する
+        break;
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)